.github/workflows/tests.yml

```
@@ -13,8 +13,16 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.2, 7.3, 7.4]
-        laravel: [^7.0]
+        php: [7.4, 7.3, 7.2.5]
+        laravel: [8.*, 7.*]
```

```suggestion
        laravel: [^8.0, ^7.0]
```

we use caret version range here